src/context/dependencies-plan.js

```diff
@@ -24,5 +24,6 @@ module.exports = plan =>
         .addModule('openIdClient', () => require('openid-client'))
         .addModule('os', () => require('os'))
         .addModule('superagent', () => require('superagent'))
-        .addUsingFunction('diffString', () => require('json-diff').diffString),
+        .addUsingFunction('diffString', () => require('json-diff').diffString)
+        .addModule('lodash', () => require('lodash')),
```

Comment:
If this comes to the context, every usage of lodash should use the context as-well. 

A [PR](https://github.com/ForestAdmin/toolbelt/pull/546/files) is ongoing for that so let's skip

Reply:
👍